packages/table/components/CheckboxTable.tsx

```diff
@@ -7,7 +7,11 @@ export interface CheckboxTableProps extends TableProps {
   /**
    * Callback for when a user makes a selection. Passes selected row data as a parameter
    */
-  onChange?: (selectedRows: object[]) => void;
+  onChange?: (selectedRows: string[]) => void;
+  /**
+   * The unique key in each row's data object
```

Comment:
Is there a strong reason to have `selectedRows` on the `TableProps` and not on `CheckboxTableProps`?

Reply:
Why `Table` should be concerned about `selectedRows`?

Reply:
`Table` needs to know which rows are selected for styling purposes. When a row is selected, it has a light grey background

Reply:
Can this info be transferred differently? Like a method `CheckboxTable` would override?

Reply:
Sure, I can give that a try.

Reply:
I tried writing a method in Table that gets overridden in CheckboxTable to handle this, but I couldn't get it to behave as expected.
If this is important to fix before merging, maybe I could show you what I tried

Reply:
I got around this with using theming, per Philipp's suggestion 🎉